test/CMakeLists.txt

```diff
@@ -4,6 +4,11 @@
 
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/CTestCustom.cmake ${CMAKE_BINARY_DIR}/CTestCustom.cmake)
 
+# Temporary - needs to be replaced with something better
+if (AMR_WIND_TEST_WITH_PYTHON)
+  set(PYTHON_REQUIREMENTS "numpy netCDF4 matplotlib pathlib pandas")
```

Comment:
This should be removed. CMake should only be checking for dependencies and not fulfilling them. Also CMake can find Python and numpy I think https://cmake.org/cmake/help/latest/module/FindPython.html but not much beyond that. Maybe there are good ways for CMake to check for Python packages, like `pip freeze` or something.

Reply:
got inspiration from here: https://stackoverflow.com/questions/62553007/cmake-check-python3-and-python-packages-presence

decided that warnings are sufficient; wouldn't want to fail to compile because a post-processing script test won't pass.